src/csharp/Grpc.Core/Internal/AsyncCall.cs

```diff
@@ -494,13 +494,13 @@ private INativeCall CreateNativeCall(CompletionQueueSafeHandle cq)
                 return injectedNativeCall;  // allows injecting a mock INativeCall in tests.
             }
 
-            var parentCall = details.Options.PropagationToken != null ? details.Options.PropagationToken.ParentCall : CallSafeHandle.NullInstance;
+            var parentCall = details.Options.PropagationToken != null ? details.Options.PropagationToken.AsImplOrNull().ParentCall : CallSafeHandle.NullInstance;
```

Comment:
nit: `PropagationToken.AsImplOrNull().ParentCall` is a potential null access?

Reply:
fixed.